plugins/combattagplus-paper/src/main/java/net/minelink/ctplus/nms/NpcNetworkManager.java

```diff
@@ -1,40 +1,12 @@
 package net.minelink.ctplus.nms;
 
 import net.minecraft.network.Connection;
-import net.minecraft.network.PacketListener;
 import net.minecraft.network.protocol.PacketFlow;
 import net.minelink.ctplus.util.EmptyChannel;
 
-import java.lang.reflect.Field;
-
 public class NpcNetworkManager extends Connection {
     public NpcNetworkManager() {
         super(PacketFlow.SERVERBOUND);
         channel = new EmptyChannel(null);
     }
-
-    @Override
-    public void setListener(PacketListener packetListener) {
```

Comment:
Is it safe to remove this?

Reply:
Yeah, Mojang changed the way it's handled so we don't have to care about it anymore.